src/NServiceBus.Core/Sagas/InvokeSagaNotFoundBehavior.cs

```diff
@@ -22,11 +24,35 @@ class InvokeSagaNotFoundBehavior : Behavior<IIncomingLogicalMessageContext>
 
             logger.InfoFormat("Could not find a started saga for '{0}' message type. Going to invoke SagaNotFoundHandlers.", context.Message.MessageType.FullName);
 
-            foreach (var handler in context.Builder.BuildAll<IHandleSagaNotFound>())
+            var nullTaskExceptions = new List<InvalidOperationException>();
```

Comment:
I'm wondering if it's worth the extra effort here to invoke all the handlers and collect the exceptions vs. just throwing the `InvalidOperationException` on the first null task?